bluesky/plans.py

```diff
@@ -92,9 +92,12 @@ def _pre(self):
         # itself as its argument and returns a generator of messages
         if self.pre_run is not None:
             yield from self.pre_run(self)
+        staged = []  # to track duplicates
```

Comment:
Yes. It is a local variable that gets discarded. But first, it tracks duplicates.